pybossa/syncer/project_syncer.py

```diff
@@ -81,6 +81,17 @@ def _build_payload(self, project, target=None):
 
         if not target:
             payload = self._remove_reserved_keys(project_dict)
+            input_data = output_data = ""
+            if "data_classification" in project.info and "input_data" in project.info["data_classification"]:
+                input_data = project.info["data_classification"]["input_data"]
+            if "data_classification" in project.info and "output_data" in project.info["data_classification"]:
+                output_data = project.info["data_classification"]["output_data"]
```

Comment:
Will the following do the same jobs for those 5 lines?
```
input_data = project.info.get("data_classification", {}).get("input_data", "")
output_data = project.info.get("data_classification", {}).get("output_data", "")
```

Reply:
Hi Xi, I was getting an error that project.info has no .get method.. so I used this instead